### DISCUSSION TITLE
How to change login email to Supabase dashboard?

### DISCUSSION THREAD

**Discussion by the0star** [supabase/supabase#23375] [Questions] [answered]
How to change login email to Supabase dashboard?

**Comment by GaryAustin1** [comment]
You would need to contact support.
Edit:  It appears there now some suggested ways:

**Reply by GaryAustin1** [reply]
Sorry this discusses it:
https://github.com/orgs/supabase/discussions/21147

**Reply by the0star** [reply]
If we didn't link GitHub and used email + password, then we have to contact support? (I didn't login with GitHub at first, and there seems to be no way to link it after.)

**Reply by GaryAustin1** [reply, marked as answer]
![image](https://github.com/supabase/supabase/assets/54564956/cb37fd53-f109-422a-b53f-9295e2caced3)
Seems you can.
I am just providing you the info Supabase has published.

**Reply by the0star** [reply]
Looks like I have to create a new account. Thanks for the info.
